code/src/App.js

```diff
@@ -1,9 +1,21 @@
 import React from 'react';
+import { BrowserRouter, Route, Routes, Navigate } from 'react-router-dom';
+// import Header from 'components/Header';//
```

Comment:
Consider looking through the code one last time before sending it in, and remove these kinds of unnecessary comments (: